src/plugins/wb-randomize/wb-randomize.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @title WET-BOEW Randomize
+ * @overview This plugin randomly picks one of the child component to be shown on the browser.
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @masterbee
+ */
+
+( function( $, wb ) {
+"use strict";
+
+$( ".provisional[data-wb-randomize]" ).each( function() {
+	var $elm = $( this ),
+		command = $elm.data( "wb-randomize" ),
+		shuffle = command.shuffle,
+		selector = command.selector,
+		number = command.number,
+		context = $elm,
+		selected = $( selector, context );
+
+	selected.shuffle = wb.shuffle;
+
+	if ( shuffle ) {
+		selected = selected.shuffle();
+	}
+
+	selected.rand = wb.rand;
+	selected.rand( number ).toggleClass( "hide" );
```

Comment:
It always toggle? I though it was either shuffle or toggle or "shuffle and toggle"

Reply:
so what shuffle does is to shuffle the order of child components. For example, if I have img1, img2 and img3 in order and if I use shuffle, the order will be mixed. What toggle does is to select a certain number of child elements.

Reply:
I guess if it were to only mix the order, toggle should not be default. Please let me know what your opinion is. I will fix accordingly.